### PR TITLE
Add latest PHP versions (Revises #473)

### PR DESCRIPTION
**Pull request by hnw**
See #473 .

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -43,6 +43,9 @@ before_install:
         if [[ "$DEFINITION" == "5.2.17" ]]; then
             brew install mysql
         fi
+        if [[ "$DEFINITION" == "7.2.0RC1" ]]; then
+            brew upgrade pkg-config
```
Comment (GrahamCampbell) [inline .travis.yml]: Why not always upgrade this?

Comment (hnw) [reply .travis.yml]: As I wrote in #473, this is a workaround and to be removed in PHP 7.2.0RC2.

**Comment by Majkl578** [comment]
@GrahamCampbell Hi, is there anything blocking 7.2.0RC1 to be rolled out? There appears to be an ABI break in RC1 causing issues with extensions (Mongo at least - the fix breaks <RC1 compatibility and Mongo-related are now failing on beta3, see https://github.com/mongodb/mongo-php-driver/issues/649). Thanks.

**Comment by jmikola** [comment]
@Majkl578: That may be a question for https://github.com/travis-ci/php-src-builder, which uses this project. I've asked that project maintainer on [Twitter](https://twitter.com/jmikola/status/907238814128766977) about an ETA.

**Pull request merged**